.gitignore

```diff
@@ -31,3 +31,7 @@ stdout/
 work-dpi/
 tb/riscv-isa-sim/
 work-*/*
+deps/fpu/src/common_cells*
+deps/fpu/src/fpu_div_sqrt_mvp*
+deps/fpu/tb*
```

Comment:
Why do we ignore those files?

Reply:
These folders are git submodules inside `fpnew`, which get automatically checked out with bender.

Reply:
`common_cells` and `fpu_div_sqrt_mvp` are checked out separately in `deps/`

Reply:
Is there a reason to:
1. Don't check them in?
2. I think you can disable the recursive checkout in the `Bender.yml`, please help @fabianschuiki

Reply:
Recursive checkout disable is currently not supported by bender.

Reply:
The checked out submodules get checked out by bender anyways, except the test bench of `fpnew`, which  does not matter in the context of Ariane.